### PR TITLE
Fix multi-level indentation issue on home page

### PR DESCRIPTION
**Pull request by anfredette**
Though 2 spaces seem to work in a lot of places, and my markdown linter complains about indenting by 4 spaces, mkdocs is apparently looking for 4 spaces to define a sub-bullet.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -17,133 +18,138 @@ monitoring, tracing and security.
 Demand is increasing from both Kubernetes developers and users. Examples of eBPF
 in Kubernetes include:
 
-- [Cilium](https://cilium.io/)
-  [Calico](https://www.tigera.io/project-calico/) CNIs 
+- [Cilium](https://cilium.io/) [Calico](https://www.tigera.io/project-calico/)
```
Comment (Billy99) [inline docs/index.md:21]:
```suggestion
- [Cilium](https://cilium.io/) and [Calico](https://www.tigera.io/project-calico/)
```

**Review by Billy99 — COMMENTED** [review]
I hate not merging over one word, but I think there needs to be a separator between the two CNI names.

File: docs/index.md (hunk 1 above)
Comment (anfredette) [reply docs/index.md]: Agreed.

**Review by Billy99 — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**